### PR TITLE
[Google Blockly] Create wrapper class for Blockly

### PR DESCRIPTION
**Pull request by ajpal**
This is hopefully the first step of migrating to [Google's Blockly](https://github.com/google/blockly) instead of [Our Blockly](https://github.com/code-dot-org/blockly).

This PR doesn't load the new Blockly, it just creates a wrapper around our Blockly. Right now, Blockly is a globally-accessible object and we get/set properties throughout our codebase. My general strategy here is to use wrapper classes to unify the interface for our blockly and google's blockly. Then I can make it dynamically decide which Blockly to load, enabling us to switch over one lab at a time.

Next step: Begin the wrapper class for Google's Blockly and make Flappy work with the google blockly wrapper

[Usage of Blockly in our codebase](https://docs.google.com/spreadsheets/d/17c0VRX9GV9SKTxbfKPoPmlSLWT5UoWzsGroSts3wuzE/edit)
[doc](https://docs.google.com/document/d/1UG-TSEfdMglzOmZbvWeyNXtUm3YfEborpRIWHgHlBjA/edit)

### CODE REVIEW

File: apps/src/sites/studio/pages/cdoBlocklyWrapper.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+const BlocklyWrapper = function(blocklyInstance) {
+  this.blockly_ = blocklyInstance;
+  this.Msg = blocklyInstance.Msg;
```
Comment (maddiedierker) [inline apps/src/sites/studio/pages/cdoBlocklyWrapper.js]: nit: should this line be `this.Msg = this.blockly_.Msg;` for consistency?

File: apps/src/sites/studio/pages/cdoBlocklyWrapper.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  blocklyWrapper.wrapReadOnlyProperty('RTL');
+  blocklyWrapper.wrapReadOnlyProperty('SVG_NS');
+  blocklyWrapper.wrapReadOnlyProperty('tutorialExplorer_locale');
+  blocklyWrapper.wrapReadOnlyProperty('useContractEditor');
+  blocklyWrapper.wrapReadOnlyProperty('useModalFunctionEditor');
+  blocklyWrapper.wrapReadOnlyProperty('Variables');
+  blocklyWrapper.wrapReadOnlyProperty('weblab_locale');
+  blocklyWrapper.wrapReadOnlyProperty('Xml');
+
+  blocklyWrapper.wrapSettableProperty('assetUrl');
```
Comment (maddiedierker) [inline apps/src/sites/studio/pages/cdoBlocklyWrapper.js:93]: [out of curiosity] how did you know which properties should be readonly vs read-write? did you just check the usage in our codebase?

Comment (ajpal) [reply apps/src/sites/studio/pages/cdoBlocklyWrapper.js:93]: There definitely might've been a better way to do this haha, but I grepped for "Blockly." and "blockly." in the whole codebase, put that into [this spreadsheet](https://docs.google.com/spreadsheets/d/17c0VRX9GV9SKTxbfKPoPmlSLWT5UoWzsGroSts3wuzE/edit#gid=1456923540), then filtered for rows that contained `=` to find the properties that we set (captured in the Setting Properties tab of that sheet)

Comment (maddiedierker) [reply apps/src/sites/studio/pages/cdoBlocklyWrapper.js:93]: lol sometimes that's the best way to do it

**Review by maddiedierker — APPROVED** [review]
left a tiny nit comment, but this looks great! so exciting to see this work start happening 🎉

**Comment by joshlory** [comment]
This is great! ✨

**Pull request merged**